toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -65,13 +56,9 @@
         "shim",
         "sqlite",
         "systemd",
-        "tar",
         "tdnf",
-        "usbutils",
         "util-linux",
         "veritysetup",
-        "vim",
-        "wget",
```

Comment:
In the runliveinstaller script, we have implementation for PXE boot support: https://github.com/microsoft/CBL-Mariner/blob/32cf2a208bddb0bd9d3f5fdaac541ed6b20a2d6b/toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller#L72. And within this implementation, we're actually calling wget. Since we're removing PXE boot support from the default ISO and plan to add new ISO or instructions for PXE boot scenario, does it still make sense to keep this implementation within our runliveinstaller? For PXE boot, should we consider using a PXE-specific installer script?

Reply:
By default it looks for default ISO else if look for imageconfig command. For this default ISO it will not go to else if and when customers use PXE boot supported ISO this is needed. Hence we can keep it this way.